sql-native/src/main/java/org/ballerinalang/sql/utils/RecordIteratorUtils.java

```diff
@@ -1,5 +1,5 @@
 /*
- *  Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *  Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
No need to change the license year. It should be the year when the file first added.

Reply:
Noted. will change that